src/main.rs

```diff
@@ -84,12 +84,12 @@ fn ws_get_base_socket(socket: &SOCKET) -> SOCKET {
             == WSAIoctl(
                 *socket,
                 SIO_BASE_HANDLE,
-                0 as *mut _,
+                NULL,
                 0,
                 &mut base_socket as *mut _ as LPVOID,
                 size_of::<SOCKET>() as DWORD,
                 &mut bytes as *mut _,
-                0 as *mut _,
+                NULL as _,
```

Comment:
This is probably fine, but `std::ptr::null_mut()` might work as well.

Reply:
I would opt to stay w/ `NULL` defined in the `winapi` crate here... just more of an FYI.

Reply:
That's nice. I' ll keep that in mind.